sympy/functions/special/gamma_functions.py

```diff
@@ -160,6 +160,13 @@ def _eval_is_real(self):
         if x.is_positive or x.is_noninteger:
             return True
 
+    def _eval_is_positive(self):
+        x = self.args[0]
+        if x.is_positive:
+            return True
+        elif x.is_noninteger:
```

Comment:
@smichr 
Can you tell me:
What's the best way to add XFAIL within a test/function,
Example:
```python
def test_foo_bar():
    assert something1
    assert something2    #this line is expected to fail, how to add XFAIL for this?
    assert something3
```